contractsSol5/KyberNetwork.sol

```diff
@@ -885,10 +888,12 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils4, IKyberNetwork, Reentra
                 (tradeData.input.dest != ETH_TOKEN_ADDRESS),
             hint
         );
-        reservesData.isFeeAccountedFlags = kyberStorage.getFeeAccountedData(reservesData.ids);
+        (reservesData.isFeeAccountedFlags, reservesData.isEntitledRebateFlags) = kyberStorage
```

Comment:
can u fix ugly prettier?

Reply:
Not sure what's the ideal style here... see if it's ok in update